avalon/__main__.py

```diff
@@ -29,8 +29,8 @@
         loader.show(debug=True)
 
     elif args.manager:
```

Comment:
Don't forget to rename the argument too.

Reply:
Fixed with: https://github.com/getavalon/core/pull/388/commits/bf4c26907522a04ec77274d8f862e853a64f7d6a